script.js

```diff
@@ -0,0 +1,2 @@
+const targetElement = document.getElementById('list');
+targetElement.insertAdjacentHTML('afterbegin','<li>これです</li>');
```

Comment:
MUST
Let's insert a blank line here as well.

Reply:
Thank you for informing me very mush!
I set the editor.

Insert a blank line at the end of the sentence.